{
  try {
    mFailureQueue.put(failure);
  }
 catch (  InterruptedException e) {
    throw new UnsupportedOperationException("Can't handle InterruptedException in onCaptureFailed");
  }
}
